ab2d-filters/src/test/java/gov/cms/ab2d/filter/EOBLoadUtilitiesTest.java

```diff
@@ -167,27 +186,55 @@ public void testItems() throws ParseException {
     @Test
     void testReaderEOB() throws IOException {
         ClassLoader classLoader = Thread.currentThread().getContextClassLoader();
-        try (InputStream inputStream = classLoader.getResourceAsStream("eobdata/EOB-for-Carrier-Claims.json")) {
-            try (Reader reader = new java.io.InputStreamReader(inputStream, StandardCharsets.UTF_8)) {
-                assertNull(EOBLoadUtilities.getEOBFromReader((Reader) null, context));
-                // STU3
-                ExplanationOfBenefit benefit = (ExplanationOfBenefit) EOBLoadUtilities.getEOBFromReader(reader, context);
-                assertNotNull(benefit);
-                assertEquals("Patient/-199900000022040", benefit.getPatient().getReference());
-            } catch (Exception e) {
-                e.printStackTrace();
-            }
-        } catch (Exception ex) {
-            ex.printStackTrace();
-        }
+
+        // DSTU3
+        InputStream inputStream = classLoader.getResourceAsStream("eobdata/EOB-for-Carrier-Claims.json");
```

Comment:
add the file paths to constants? but I do understand if you dont want to since they are just being used few times in here

Reply:
I'd rather keep it as is, if that's okay!